apple-pie/styles/style.css

```diff
@@ -0,0 +1,68 @@
+
+* {
+	box-sizing: border-box;
+	margin: 0 10%;
```

Comment:
To get rid of the white bars around the background image change your margin here to this `margin: 0px;`

After this you can adjust your left margins on the text to center them a little bit more.